Dockerfile

```diff
@@ -19,7 +19,7 @@ COPY web/nginx.conf /etc/nginx/nginx.conf
 RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
 
 RUN composer install
-RUN echo "" > /app/storage/db.sqlite
+RUN touch /app/storage/db.sqlite
```

Comment:
This would wipe the db on every deploy, but now it'll copy it from the current instance if there is one. In general, production apps shouldn't be using sqlite so this wouldn't typically happen in a real world scenario.